fake-snippets-api/routes/api/order_quotes/create.ts

```diff
@@ -7,17 +7,19 @@ export default withRouteSpec({
   jsonBody: z.object({
     package_release_id: z.string(),
     vendor_name: z.string(),
+    circuit_json: z.any().optional(),
   }),
   jsonResponse: z.object({
     order_quote_id: z.string(),
   }),
 })(async (req, ctx) => {
-  const { package_release_id, vendor_name } = req.jsonBody
+  const { package_release_id, vendor_name, circuit_json } = req.jsonBody
 
   const orderQuoteId = ctx.db.addOrderQuote({
     account_id: ctx.auth.account_id,
     package_release_id,
     vendor_name,
+    circuit_json,
```

Comment:
if circuit json is given, then package_release_id isn't required. either circuit json or package release id is required